src/components/NewUserForm.vue

```diff
@@ -29,4 +75,7 @@
         width: 70%;
         margin: 0 auto
     }
+    div.control.input-field.select::after {
+        display: none;
+    }
```

Comment:
This is to remove the extra arrow from the select input. It was being added by the component library.
<img width="1109" alt="Screen Shot 2020-01-31 at 6 05 53 PM" src="https://user-images.githubusercontent.com/49662828/73584394-4b08d380-4455-11ea-98dc-49e2a866d2ae.png">